source/docs/articles/sites/code/reading-pantheon-environment-configuration.md

```diff
@@ -89,3 +89,146 @@ Place [Domain Access setup routine](http://drupal.org/node/1096962) at the **en
 Pantheon's default `wp-config.php` includes code to read from the `$_ENV` superglobal so no additional configuration should be required.
 
 For more information, see [configuring wp-config.php](/docs/articles/wordpress/configuring-wp-config-php).
+
+
+## Using $_SERVER
+When incorporating custom configurations on Pantheon, use `$_ENV` instead of `$_SERVER` wherever possible. `$_SERVER` is generally unavailable when executing code via the command line (e.g. [Terminus](/docs/articles/local/cli), Drush, or WP-CLI), which can cause failures for things like clearing cache. The few exceptions include `HTTP_HOST` and `REMOTE_ADDR`, or things pertaining directly to the web request in progress such as [redirects](/docs/articles/sites/code/redirect-incoming-requests/).
+
+For debugging modules or plugins, it may be beneficial to review the values in the `$_SERVER` variable versus the value used by the extension.

```

Comment:
The statement was adapted from Tara's original contribution: "If a module or plugin is not behaving as expected, reviewing this output versus the value the module/plugin is using may be beneficial."

"the extension" refers to the "module/plugin" in question when debugging, since it's unclear I'll revise it.